pyrightconfig.json

```diff
@@ -1 +1,4 @@
-{"include": ["mentat"], "ignore": ["testbed", "tests"], "typeCheckingMode": "strict"}
+{
+  "include": ["mentat", "tests", "testbed"],
+  "strict": ["mentat"]
```

Comment:
I don't think we should disable all type checking for tests/testbed modules, so I re-enabled it

Reply:
I think we should keep it disabled; for the testbed, we want to be able to make tests that don't have typing in them, and for the regular tests, it will just add the hassle of specifying the type of fixtures as argument functions.